### DISCUSSION TITLE
Please run `./vendor/bin/pest` instead of `/vendor/bin/phpunit`.

### DISCUSSION THREAD

**Discussion by ivandokov** [pestphp/pest#476] [Q&A]
I'm trying to install a package with composer but I get the following error when the code reaches the auto discovery command:

```
composer require spatie/laravel-route-discovery

Using version ^1.0 for spatie/laravel-route-discovery
./composer.json has been updated
Running composer update spatie/laravel-route-discovery
Loading composer repositories with package information
Updating dependencies
Lock file operations: 1 install, 0 updates, 0 removals
  - Locking spatie/laravel-route-discovery (1.0.0)
Writing lock file
Installing dependencies from lock file (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Downloading spatie/laravel-route-discovery (1.0.0)
  - Installing spatie/laravel-route-discovery (1.0.0): Extracting archive
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan package:discover --ansi


  Please run `./vendor/bin/pest` instead of `/vendor/bin/phpunit`.


Script @php artisan package:discover --ansi handling the post-autoload-dump event returned with error code 1
```

The package I install doesn't really matter as I tried with a few others and the error always appears.

My code is split into domains (including the tests) and I just updated to Laravel 9.
I rolled back my code to a state where the code base was using Laravel's conventions and the package installation was successful. I am not sure how moving the tests to non-standart locations cause this error.

Basically I have a testsuite for each domain in phpunit.xml

```xml
<testsuite name="Clients">
    <directory suffix="Test.php">./app/Domains/Clients/tests</directory>
</testsuite>
<testsuite name="Search">
    <directory suffix="Test.php">./app/Domains/Search/tests</directory>
</testsuite>
```

Any clues?

**EDIT:**
Reinstalling pest and the plugins I have kind of resolved the issue. I can again install packages but one specific package still brings the issue - `spatie/laravel-event-sourcing` and maybe I have to talk to Spatie guys about this issue.

Here is a stack trace from the exception https://flareapp.io/share/LPdEQMN7#F28
